src/axom/quest/examples/CMakeLists.txt

```diff
@@ -39,13 +39,12 @@ if(ENABLE_MPI AND MFEM_FOUND AND MFEM_USE_MPI
 endif()
 
 # Distributed closest point example -------------------------------------------
-if(ENABLE_MPI AND MFEM_FOUND AND MFEM_USE_MPI
-              AND AXOM_ENABLE_SIDRE AND AXOM_ENABLE_MFEM_SIDRE_DATACOLLECTION)
+if(ENABLE_MPI AND AXOM_ENABLE_SIDRE)
```

Comment:
Are mfem meshes still supported as input to the distributed closest point example?

Reply:
No.  I thought the effort of preserving it during the current changes was more than re-implementing it afterwards.